pages/attacks/Content_Spoofing.md

```diff
@@ -51,6 +53,13 @@ followers and distribute misleading Content Spoofing payload via
 Twitter/Facebook/Instagram/ similar popular channel. This will lead
 media to assume news is correct and create headline stories.
 
+## Audit Guideline
+
+Text injection can be easily found where
+- User input via parameter or directly in the URL is reflected in the page response.
```

Comment:
if

Reply:
Should be consistent in using (or not) periods for bullets

Reply:
is it fine?